GitHub Actions/haal-centraal.md

```diff
@@ -138,4 +138,7 @@ De workflow voert de volgende stappen uit:
 [6]: https://openapi-generator.tech/
 [7]: https://github.com/postmanlabs/openapi-to-postman
 [8]: https://github.com/marketplace/actions/select-matching-issues
-[9]: ./haal-centraal-npm-beheer.md
\ No newline at end of file
+[9]: ./haal-centraal-npm-beheer.md
+
+Deze workflow maakt gebruik van de secret USER_TOKEN, een ge-encrypte omgevingsvariabele waarin acces tokens zijn vastgelegd waarmee autorisatie verkregen kan worden voor het aanbrengen van wijzigingen op de repository.
```

Comment:
Dit klopt niet meer, want nu wordt de GITHUB_TOKEN gebruikt.

Reply:
Goed opgemerkt, is aangepast.